DESCRIPTION

```diff
@@ -55,7 +55,8 @@ Suggests:
     shinystan (>= 2.3.0),
     survival,
     testthat (>= 2.0.0),
-    vdiffr (>= 1.0.2)
+    vdiffr (>= 1.0.2),
+    ggdist
```

Comment:
Not a big deal, but can you move ggdist up in the list so that it's in alphabetical order like all the other packages listed?